docs/BreakingChanges/082 WorkflowDesigner.Load doesn't remove symbol property.md

```diff
@@ -0,0 +1,27 @@
+## 82: WorkflowDesigner.Load doesn't remove symbol property
```

Comment:
{
  "Id": "82",
  "Title": "WorkflowDesigner.Load doesn't remove symbol property",
  "Details": "When targeting the .NET Framework 4.5 in the workflow designer, and loading a re-hosted 3.5 workflow with the WorkflowDesigner.Load() method, a XamlDuplicateMemberException is thrown while saving the workflow.",
  "Suggestion": "This bug only manifests when targeting .NET Framework 4.5 in the workflow designer, so it can be worked around by setting the `WorkflowDesignier.Context.Services.GetService<DesignerConfigurationService>().TargetFrameworkName` to the 4.0 .NET Framework.\n\nAlternatively, the issue may be avoided by using the [WorkflowContext.Load(string)](https://msdn.microsoft.com/en-us/library/ee425926(v=vs.110).aspx) method to load the workflow, instead of [WorkflowDesigner.Load()](https://msdn.microsoft.com/en-us/library/ee403482()v=vs.110).aspx).",
  "Link": "https://connect.microsoft.com/VisualStudio/feedback/details/786503/",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Activities.Presentation.WorkflowDesigner.Load"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5.2",
  "IsBuildTime": false,
  "IsQuirked": true,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": true,
  "ImpactScope": 1
}

**Parsed APIs:**
- [x] M:System.Activities.Presentation.WorkflowDesigner.Load